packages/core/src/components.d.ts

```diff
@@ -35,6 +36,7 @@ declare global {
 declare namespace LocalJSX {
     interface DsoAlert {
         "level"?: 'success' | 'info' | 'warning' | 'danger';
```

Comment:
Waarom is eigenlijk hier `level` optioneel, terwijl dat in `Components` niet is?

Reply:
`components.d.ts` wordt door de Stencil.js compiler gegenereerd. Het was mij ook al opgevallen maar ik heb er nog geen onderzoek naar gedaan waarom Stencil.js dit doet

Reply:
https://stenciljs.com/docs/typed-components#required-properties

Misschien dat dit de oplossing is. Dat zou wel mooi zijn, want dan kan `strict: true` in `tsconfig.json` aan :tada:

Reply:
Zeker bijzonder gedrag, maar door de prop required te maken is dit issue wel opgelost.